tests/testthat/test-if2.R

```diff
@@ -0,0 +1,143 @@
+context("IF2")
+
+test_that("Can run IF2", {
```

Comment:
could you add a test that shows it converges on approximately the right likelihood for the volatility case?